Minigames/src/main/java/au/com/mineauz/minigames/commands/HelpCommand.java

```diff
@@ -57,11 +57,11 @@ public boolean onCommand(CommandSender sender, Minigame minigame,
         if (sender instanceof Player) {
             player = (Player) sender;
         }
-        sender.sendMessage(ChatColor.GREEN + "List of Minigame commands");
+        sender.sendMessage(ChatColor.GREEN + MessageManager.getUnformattedMessage(null, "command.info.header"));
         sender.sendMessage(ChatColor.BLUE + "/minigame");
-        sender.sendMessage(ChatColor.GRAY + MessageManager.getMessage(null, "command.info.header"));
+        sender.sendMessage(ChatColor.GRAY + MessageManager.getUnformattedMessage(null, "command.info.mgm"));
         if (player == null || player.hasPermission("minigame.join")) {
-            sender.sendMessage(ChatColor.BLUE + MessageManager.getMessage(null, "command.info.mgm"));
+            sender.sendMessage(ChatColor.BLUE + "/minigame join <Minigame>");
```

Comment:
This seems to be reverting the use of MessageManager to a literal string, which we want to move away from.. was this a mistake?